java/client/src/main/java/glide/managers/RedisExceptionCheckedFunction.java

```diff
@@ -2,23 +2,24 @@
 package glide.managers;
 
 import glide.api.models.exceptions.RedisException;
+import response.ResponseOuterClass.Response;
```

Comment:
Please revert this file.  Javadoc standards for internal/private files are not the same, and it's fine as is.

Reply:
Reverted